app/celer-g4/test-harness.py

```diff
@@ -116,6 +116,7 @@ def strtobool(text):
     json.dump(inp, f, indent=1)
 
 print("Running", exe, inp_file, file=stderr)
+print(f"To run directly use: {exe} {problem_name}.inp.json", file=stderr)
```

Comment:
A completely related question.  What is the advantage of using a pipe rather than passing the file name?

> Also you should print the working directory as part of the "how to run directly".

Yes we could but then to be complete we should also print the environment variable ... and at that point we may as well generate a bash script :).   

More to the point the current output of `ctest -V` contains everything that is needed to run by hand (with minor tweak to be done during the copy/pasting) .... except for the name of the json file ... or for the fact that this json file even exist :)

```
270: Working Directory: /afs/cern.ch/user/p/pcanal/atlas-on-eos/celeritas/gcc13-local/build/celeritas/app/celer-g4
270: Environment variables:
270:  G4NEUTRONHPDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4NDL
270:  G4LEDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4EMLOW
270:  G4LEVELGAMMADATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/PhotonEvaporation
270:  G4RADIOACTIVEDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/RadioactiveDecay
270:  G4PARTICLEXSDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4PARTICLEXS
270:  G4PIIDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4PII
270:  G4REALSURFACEDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/RealSurface
270:  G4SAIDXSDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4SAIDDATA
270:  G4ABLADATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4ABLA
270:  G4INCLDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4INCL
270:  G4ENSDFSTATEDATA=/cvmfs/atlas-nightlies.cern.ch/repo/sw/local/simulation/main_AthSimulation_x86_64-el9-gcc13-opt/Geant4/share/geant4.11.1.patch03.atlasmt02/data/G4ENSDFSTATE
270:  CELER_LOG_LOCAL=debug
270:  CELER_DISABLE_PARALLEL=1
270:  CELER_DEBUG=OFF
270:  CELER_USE_ROOT=ON
270:  CELER_BUILD_TYPE=Debug
270:  G4FORCENUMBEROFTHREADS=2
270:  OMP_NUM_THREADS=1
270:  CELER_TEST_EXT=gpu
270: Test timeout computed to be: 1500
270: Running /afs/cern.ch/user/p/pcanal/atlas-on-eos/celeritas/gcc13-local/build/celeritas/bin/celer-g4 -
```